### PR TITLE
Fixes #1328. Fix error expectation places for CFE

### LINKED ISSUE #1328: Failures on [cfe] Refactor TypeBuilders
These tests need expectations updates after landing [[cfe] Refactor TypeBuilders](https://dart-review.googlesource.com/c/sdk/+/239023):

```
co19/LanguageFeatures/Super-mixins/subclassable_types_t01 MissingCompileTimeError (expected Pass)
co19/LanguageFeatures/Super-mixins/superclass_constraint_t01 MissingCompileTimeError (expected Pass)
```

### PR DESCRIPTION
**Pull request by sgrekhov**

### CODE REVIEW

File: LanguageFeatures/Super-mixins/subclassable_types_t01.dart [hunk 1]
```diff
@@ -54,6 +52,7 @@ mixin M7 on B, C implements Null {}
 // [cfe] 'Null' is restricted and can't be extended or implemented.
```
Comment (eernstg) [inline LanguageFeatures/Super-mixins/subclassable_types_t01.dart:52]: Shouldn't line 51 and 52 be deleted?

File: LanguageFeatures/Super-mixins/superclass_constraint_t01.dart [hunk 2]
```diff
@@ -48,6 +46,7 @@ mixin M7 on Null implements I {}
 //    ^
 // [cfe] 'Null' is restricted and can't be extended or implemented.
```
Comment (eernstg) [inline LanguageFeatures/Super-mixins/superclass_constraint_t01.dart:47]: Again, it looks like lines 46 and 47 should be deleted.

**Review by eernstg — COMMENTED** [review]
Added a few comments (it looks like some lines should have been deleted).

File: LanguageFeatures/Super-mixins/superclass_constraint_t01.dart (hunk 2 above)
Comment (sgrekhov) [reply LanguageFeatures/Super-mixins/superclass_constraint_t01.dart:47]: The same as the above. There are two error on this line in CFE

File: LanguageFeatures/Super-mixins/subclassable_types_t01.dart (hunk 1 above)
Comment (eernstg) [reply LanguageFeatures/Super-mixins/subclassable_types_t01.dart:52]: OK, thx!

**Review by eernstg — APPROVED** [review]
LGTM

**Pull request merged**